eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -4,7 +4,12 @@ schedules:
   branches:
     include:
     - main
-    - release/*
+    - release/*.0.1xx*
+    - internal/release/*.0.1xx*
+    exclude:
+    # These 8.0.1xx-* branches can be removed from the exclusions once they get deleted from the repo

```

Comment:
As part of https://github.com/dotnet/source-build/issues/3407, I thought we had agreement to remove the preview branches as part of this work so that these exclusions would not be necessary?

Reply:
Followed up on that here: https://github.com/dotnet/source-build/issues/3407#issuecomment-1767162435